tests/integration/render.rs

```diff
@@ -2,16 +2,21 @@ use assert_cmd::prelude::*;
 use cmd_lib::run_fun;
 use std::fs::OpenOptions;
 use std::io::Read;
-use std::io::{self, Write};
+use std::io::Write;
 use std::path::PathBuf;
 use std::process::Command;
 
 const BIN_NAME: &'static str = env!("CARGO_PKG_NAME");
 
-/// Generate a dummy configuration and dump it to a temporary location on disk
-/// this should be called at the beginning of each test case and concluded with calling `drop_temp_folder`.
-fn generate_config() -> Result<PathBuf, io::Error> {
-    let config = r#"---
+struct Config {
```

Comment:
@Kenec I wrapped the unique config into a struct and implemented the `Drop` trait on it. That way, Rust will call it automatically when the config gets out of scope and therefore a tear-down is implemented. No need to call any drop function.